server/send-emails.js

```diff
@@ -0,0 +1,98 @@
+require('dotenv/config');
+const pg = require('pg');
+const nodemailer = require('nodemailer');
+
+const db = new pg.Pool({
+  connectionString: process.env.DATABASE_URL,
+  ssl: {
+    rejectUnauthorized: false
+  }
+});
+
+const transporter = nodemailer.createTransport({
+  host: 'smtp.gmail.com',
+  port: 465,
+  secure: true,
+  auth: {
+    user: process.env.USER,
+    pass: process.env.PASSWORD
+  }
+});
+
+let sql = `
+  select (recipient, content, "capsuleId")
+    from capsules
+   where ("sendOn" < now() AND "sentAt" IS NULL)
+`;
+db.query(sql)
+  .then(res => {
+    for (let i = 0; i < res.rows.length; i++) {
+      const rowAsArr = res.rows[i].row.split(',');
+      const recipient = rowAsArr[0].substring(1);
+      const location = rowAsArr[1];
+      const capsuleId = rowAsArr[2].substring(0, rowAsArr[2].length - 1);
```

Comment:
I don't really understand what is going on here. Why aren't the rows just objects?